packages/cardano-services/test/StakePoolSeach/DbSyncStakePoolSearchProvider/__snapshots__/StakePoolSearchBuilder.test.ts.snap

```diff
@@ -140,8 +140,8 @@ Object {
 exports[`StakePoolSearchBuilder buildAndQuery buildAndQuery, queryPoolHashes & queryTotalCount 2`] = `
 Array [
   Object {
-    "id": "15",
-    "updateId": "309",
+    "id": "16",
```

Comment:
What changed to invalidate the snapshot?

Reply:
Snapshots failed [here](https://github.com/input-output-hk/cardano-js-sdk/runs/6507593263?check_suite_focus=true) too after merge, but the PR checks were green :thinking:

Reply:
I needed to add some tables to the dump and the database used before was a db that Juan Cruz had in his computer. The one I used had different ids (but same data) for some records, so I had to update the snapshots. We are now sharing the same one

Reply:
Can the original one be used instead @lgobbi-atix, or would that just shift the impact?

Reply:
Can we generate a snapshot from the original DB, or would that just shift the impact?

Reply:
Thinking about it with Juan Cruz maybe it's better to use a db with the [snapshots](https://updates-cardano-testnet.s3.amazonaws.com/cardano-db-sync/index.html#12/) that db-sync provides but without running db-sync.
That way all devs could have their own frozen local database (using the same snapshot version of course) and then run the dump script on it if they need to add more tables/blocks to the fixture.

There shouldn't be any issues with ids anymore if everyone is using the same snapshot and db-sync is not running.

Reply:
We also found another issue with the order of the queries' results, apparently it was not always the same. For example, running `yarn test chainhistoryhttpservice` would pass, but running `yarn test` would fail because it was expecting the same data but in a different order in the snapshot. I added `order by` to the queries and now it's working

Reply:
That should be the reason why this was happening

> Snapshots failed [here](https://github.com/input-output-hk/cardano-js-sdk/runs/6507593263?check_suite_focus=true) too after merge, but the PR checks were green

Reply:
CI now is failing because of dates, it looks like an issue with timezones. There is a 3 hour difference between expected and the snapshot, which I guess makes sense because I'm in UTC-3

Reply:
Do we have relative dates in the dataset? The DB just stores UTC. We could strip some irrelevant data like DB IDs and dates?

Reply:
It looks like the issue is that db-sync stores timestamps in UTC but without timezone (`timestamp` instead of `timestamptz`), so when making a query it assumes it's stored in the computer's timezone and tries to convert it to UTC, in my case by adding 3 extra hours.

For example, for [block 1598507](https://explorer.cardano-testnet.iohkdev.io/en/block?id=7a48b034645f51743550bbaf81f8a14771e58856e031eb63844738ca8ad72298), with time `2020/07/29 03:58:16 UTC`

If I just do `SELECT time FROM block WHERE block_no = 1598507;` I get as the `db.query` result: `time:  2020-07-29T06:58:16.000Z` (which would actually be UTC+3).
But if I specify in which timezone the `time` is by doing `SELECT time AT TIME ZONE 'UTC' FROM block WHERE block_no = 1598507;` it should always return it as `time:  2020-07-29T03:58:16.000Z`, which is correct